python_tests/test_solver.py

```diff
@@ -1,33 +1,94 @@
-from __future__ import print_function
-import sys
 import numpy as np
-import htool_complex
-import scipy.sparse.linalg as spla
+from htool import ComplexHMatrix
+from scipy.sparse.linalg import gmres
 from mpi4py import MPI
 import math
 import struct
 import os
 
-def python_gmres():
+def python_gmres_get_coef():
 
     comm = MPI.COMM_WORLD
     size = comm.Get_size()
     rank = comm.Get_rank()
 
-    scalar = np.complex128
+    # Matrix
+    with open(os.path.join(os.path.dirname(__file__)+"/../data/data_test/SPD_mat_example.bin"), "rb" ) as input:
```

Comment:
Do not use `+` to concatenate. `join("a", "b", "c")` is more portable, since it gives `a/b/c` or `a\b\c` depending on the platform. If you've already concatenated with `+`, `join` with a single argument does nothing.

Reply:
Ok thx!